drivers/sensor/mcp9600/mcp9600.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2020 SER Consulting, LLC
+ * Copyright (c) 2019 Peter Bigot Consulting, LLC
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <kernel.h>
+#include <drivers/i2c.h>
+#include <init.h>
+#include <sys/byteorder.h>
+#include <sys/__assert.h>
+#include <logging/log.h>
+#include "mcp9600.h"
+
+#define DT_DRV_COMPAT microchip_mcp9600
+
+#define MCP9600_BUS_I2C DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
```

Comment:
Does this sensor support SPI too? I don't see this macro being used anywhere

Reply:
I2C only, will check why the macro is still there, probably not needed.

Reply:
done on next PR